src/coreComponents/fileIO/silo/SiloFile.cpp

```diff
@@ -1314,8 +1315,8 @@ void SiloFile::writeElementRegionSilo( ElementRegionBase const & elemRegion,
         // check if the field actually exists / plotted on the current subregion
         if( viewPointers[esr].count( fieldName ) > 0 )
         {
-          Wrapper< arrayType > const & sourceWrapper = dynamicCast< Wrapper< arrayType > const & >( *(viewPointers[esr][fieldName]));
-          traits::ViewTypeConst< arrayType > const sourceArray = sourceWrapper.reference().toViewConst();
+          WrapperType const & sourceWrapper = dynamic_cast< WrapperType const & >( *(viewPointers[esr].at( fieldName )) );
```

Comment:
This is the only code change. I was getting some really strange error, probably a compiler bug.